go-controller/pkg/ovn/gateway_init.go

```diff
@@ -213,6 +219,23 @@ func gatewayInit(nodeName string, clusterIPSubnet []*net.IPNet, hostSubnets []*n
 		"--", "lsp-set-type", l3GatewayConfig.InterfaceID, "localnet",
 		"--", "lsp-set-options", l3GatewayConfig.InterfaceID, "network_name=" + util.PhysicalNetworkName}
 
+	if config.Gateway.Mode == config.GatewayModeHybrid {
```

Comment:
Shouldn't this be if `hybridGatewayConfig != nil` ?

Reply:
done